README.md

```diff
@@ -88,7 +89,12 @@ S. Lim's fine-tuned BERT model):
 Install and start this NLP Sandbox tool.
 
 ```console
+<<<<<<< HEAD
+cd server
+pip install -r requirements.txt
+=======
 cd server && pip install -r requirements.txt
+>>>>>>> 65fb5e2ff66e9e2643b3883e8dd600cac99751cc
```

Comment:
This appears to be an unresolved `git merge`?

Reply:
This is now fixed.